inst/extdata/FGDC_template.mustache

```diff
@@ -4,7 +4,7 @@
     <citation>
       <citeinfo>
         {{#authors}}
-  <origin>{{.}}</origin>
+  	  	<origin>{{.}}</origin>
   {{/authors}}
         <pubdate>{{pubdate}}</pubdate>
```

Comment:
these tags are always the tags given as 'shortname' in www.fgdc.gov/csdgmgraphical/, e.g., https://www.fgdc.gov/csdgmgraphical/ideninfo/citat/citinfo/pubdate.htm, right? too bad their urls don't match up to the shortnames - it'd be cool if you could provide a link to the documentation online, but maybe it's not practical for FGDC for now